### PR TITLE
Document the existence of the prefix property in config

### PR DESCRIPTION
**Pull request by sweeney**
Prefix config exists but is not documented

### CODE REVIEW

File: lib/core.js [hunk 1]
```diff
@@ -10,6 +10,7 @@ import { promiseRetry, bind, uuid } from './util';
  * @typedef {object} CoreConfig
  * @prop {string} [key] The API key ("publishable_key_..." or "pk_...") used to authenticate with the Ravelin API.
  * @prop {string} [api] A string of the format "https://api.ravelin.net" which forms the base of API requests.
+ * @prop {string} [prefix=ravjs-] A string of the default prefix for the deviceId
```
Comment (adamroyjones) [inline lib/core.js]: Not rjs-, given the body of the function Core (specifically L44)?

Comment (sweeney) [reply lib/core.js]: Whoops!

File: README.md [hunk 2]
```diff
@@ -194,6 +194,10 @@ var rav = new Ravelin({
      * Content-Security-Policy then add the api to the connect-src directive.
      */
     // api: 'https://live.ravelin.click/',
+    /**
+     * @prop {string} [prefix=rjs-] The prefix of the generated deviceId
```
Comment (adamroyjones) [inline README.md]:
```suggestion
     * @prop {string} [prefix=rjs-] The prefix of the generated deviceId.
```

File: lib/core.js [hunk 3]
```diff
@@ -10,6 +10,7 @@ import { promiseRetry, bind, uuid } from './util';
  * @typedef {object} CoreConfig
  * @prop {string} [key] The API key ("publishable_key_..." or "pk_...") used to authenticate with the Ravelin API.
  * @prop {string} [api] A string of the format "https://api.ravelin.net" which forms the base of API requests.
+ * @prop {string} [prefix=rjs-] A string of the default prefix for the deviceId
```
Comment (adamroyjones) [inline lib/core.js]:
```suggestion
 * @prop {string} [prefix=rjs-] A string of the default prefix for the deviceId.
```

**Review by adamroyjones — DISMISSED** [review]

**Review by adamroyjones — APPROVED** [review]

**Pull request merged**